test/test_group.py

```diff
@@ -46,6 +45,12 @@ def set_and_assert_group_data(db, client, route):
     assert list(sorted(issues)) == list(sorted([issue.cve.id for issue in group.issues]))
     assert list(sorted(pkgnames)) == list(sorted([pkg.pkgname for pkg in group.packages]))
 
+    if bug_ticket == '':
+        # Assert project and product category
+        assert 'project=1&amp;product_category=13' in resp.data.decode('utf-8')
```

Comment:
This assumes the project is always 1 (core/extra) but it should handle the case properly if the package is a [community] one